lib/revision_table_access_check_filter.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+// TODO: This is a temp solution that will
+// be eventually replaced by the restrictions table.
+
+var URI = require('hyperswitch').URI;
+var mwUtil = require('./mwUtil');
+var entities = require('entities');
+
+var redirectRegEx = /<link rel="mw:PageProp\/redirect" href="\.\/([^"#]+)"/;
```

Comment:
This will remove the sections from a redirect. And that's what we need as RB doesn't support sections anyway